tests/test_level/test_level_12.py

```diff
@@ -282,6 +282,160 @@ def test_print_calc(self):
 
         self.multi_level_tester(code=code, expected=expected, max_level=17)
 
+    #
+    # forward tests
+    #
+    def test_forward_with_integer_variable(self):
+        code = textwrap.dedent("""\
+            a is 50
+            forward a""")
+        expected = HedyTester.dedent(
+            "a = 50",
+            HedyTester.forward_transpiled('a'))
+
+        self.multi_level_tester(
+            code=code,
+            expected=expected,
+            extra_check_function=self.is_turtle(),
+            max_level=18,
```

Comment:
I don't think we should specify the max level here, or it should be set to HEDY_MAX_LEVEL. If level 19 is added and it would break things, we need to know.

Reply:
I've removed both max_level=18 lines in the tests for level 12.